doc/cephfs/mount-using-fuse.rst

```diff
@@ -55,6 +55,16 @@ If you have more than one FS on your Ceph cluster, use the option
 
 You may also add a ``client_fs`` setting to your ``ceph.conf``
 
+You may specify a mapping between the local user's UID (or GID) and the remote
+server's UID (or GID) by using::
+
+    ceph-fuse --id foo --uid-mapping uid_map --gid-mapping gid_map
+
```

Comment:
if you reword the arguments to `uid_map_file_name` and `gid_map_file_name`, then it becomes pretty obvious what is expected and you need not explain it separately. However, you can then say that the expected format of the mapping is as follows ... just as you've mentioned it below.